test/documentation/raptor_doxygen_cfg.in

```diff
@@ -114,23 +113,21 @@ CITE_BIB_FILES         =
 #---------------------------------------------------------------------------
 QUIET                  = NO
 WARNINGS               = YES
-WARN_IF_UNDOCUMENTED   = YES
+WARN_IF_UNDOCUMENTED   = NO
```

Comment:
This change (and the one in `INPUT` where you remove the trailing `\`) should be enough, why change the other ones?

Reply:
I made the other changes because of:
```
warning: ignoring unsupported tag 'CREATE_SUBDIRS_LEVEL' at line 14, file Doxyfile
warning: ignoring unsupported tag 'WARN_LINE_FORMAT' at line 123, file Doxyfile
warning: ignoring unsupported tag 'INPUT_FILE_ENCODING' at line 133, file Doxyfile
warning: ignoring unsupported tag 'FORTRAN_COMMENT_AFTER' at line 150, file Doxyfile
warning: ignoring unsupported tag 'HTML_COLORSTYLE' at line 181, file Doxyfile
warning: ignoring unsupported tag 'DOT_COMMON_ATTR' at line 332, file Doxyfile
warning: ignoring unsupported tag 'DOT_EDGE_ATTR' at line 333, file Doxyfile
warning: ignoring unsupported tag 'DOT_NODE_ATTR' at line 334, file Doxyfile
warning: argument 'SYSTEM' for option CASE_SENSE_NAMES is not a valid boolean value
Using the default: NO!
warning: tag INPUT: input source 'INPUT_ENCODING' does not exist
warning: tag INPUT: input source '=' does not exist
warning: tag INPUT: input source 'UTF-8' does not exist
warning: source 'INPUT_ENCODING' is not a readable file or directory... skipping.
warning: source '=' is not a readable file or directory... skipping.
warning: source 'UTF-8' is not a readable file or directory... skipping.
```

And the `WARN_IF_UNDOCUMENTED` change because of warnings like:
`Repos/raptor/include/raptor/argument_parsing/validators.hpp:154: warning: Compound raptor::bin_validator is not documented.`

You said your doxgen version is 1.9.5 mine is 1.9.3. I thougt this would not be a big difference, but I can update mine.